apps/full-site-editing/package.json

```diff
@@ -16,14 +16,15 @@
 	},
 	"homepage": "https://github.com/Automattic/wp-calypso",
 	"scripts": {
+		"php": "cp -R *plugin *theme dist",
```

Comment:
Any reason for naming this script `php` instead of something more related to copying?

Reply:
No reasons, I ran out of good names, and that has been called `php` for a long time because of its original intent of only copying `php` files.
I'll update it into `copy-to-dist` and call it a day.